### ISSUE TITLE
how is alpha fold 3 trained

### ISSUE THREAD

**Issue by joepareti54** [google-deepmind/alphafold#1031]
This paper says that the loss function is the weighted sum of L_distogram, L_diffusion and L_confidence. But how is it implemented? taking the derivative to update the weights ? When the processing is in the trunk there is no diffusion yet which occurs later in time. When the processing is in diffusion, the trunk does not seem to execute again. Is there any backpropagation?
